src/Generate/Controller.php

```diff
@@ -105,6 +111,7 @@ protected function getOptions() {
             ['template', 't', InputOption::VALUE_OPTIONAL, 'Specify custom template'],
             ['belongs-to-many', 'btm', InputOption::VALUE_OPTIONAL, 'Specify belongs to many relations'],
             ['force', 'f', InputOption::VALUE_NONE, 'Force will delete files before regenerating controller'],
+            ['with-export', 'e', InputOption::VALUE_NONE, 'Specify export to excel'],
```

Comment:
Preklep v description, spravne to ma byt "Generate an option to Export as Excel". Premenuj vsade.